sphinx-docs/installguide/release_notes.rst

```diff
@@ -16,7 +16,12 @@ On Windows, use the ``bin\windows\kalite.bat`` in the cmd.exe prompt::
 
     bin\windows\kalite.bat manage setup
 
-When you are asked whether or not to delete your database, you should choose to keep your database!
+When you are asked whether or not to delete your database, you should choose to keep your database! You will also be prompted to download an assessment items package, or to specify the location if you have already downloaded it. If you wish to download the package and specify the location during the setup process: 
+
+* Download the assessment items package `here <http://eslgenie.com/media/assessment_item_resources.zip>`_. Save it in the same folder as the setup script. Take note of the filename, as you will need to enter it exactly!
```

Comment:
> you will need to enter it exactly

This would be nice to avoid, via the recommendation in https://github.com/learningequality/ka-lite/pull/3313/files#r26718375